README.md

```diff
@@ -69,8 +66,20 @@ Check out all related information [here](GSSoC.md)
     1. [Python](https://realpython.com/python-virtual-environments-a-primer/#why-the-need-for-virtual-environments)
     2. [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
 
-#### 2. How to set me up Docker style?
-- Make sure you have Docker and Docker-Compose (refer pre-requisite steps for installation instructions)
+------
+
+#### What are some pre-requisites? (with Docker)
+
+- StanfordCoreNLP
+  - StanfordCoreNLP has a dependency on Java 8. `java -version` should complete successfully with version 1.8 or higher.
+  - Windows- Download as a .zip file from [here](https://stanfordnlp.github.io/CoreNLP/download.html).  
+  - Linux and MacOS- Follow the instructions to download the file from [here](https://stanfordnlp.github.io/CoreNLP/download.html).  
+- Docker (if you want to set me up the Docker way)
```

Comment:
Remove the (if you want to set me up the Docker way) phrase

Reply:
Extremely sorry. That was something I oversaw. :sweat: